bin/p2-log-bridge/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"os/signal"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/square/p2/pkg/logbridge"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/version"
+)
+
+var (
+	durableLogger = kingpin.Arg("exec", "An executable that logbridge will log to with durability").Required().String()
```

Comment:
WDYT 
"An executable that logbridge will log to without dropping messages. If a write to STDIN of this program blocks, logbridge will block."